@kiva/kv-components/vue/KvCommentsListItem.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+	<div>
+		{{ text }}
```

Comment:
I see that you removed the avatar in the last commit. Please add back in, since we will have a user photo eventually, once we figure out where to source. We just need to allow for the component to fallback to hiding the avatar when it's missing, like we do in the add comment component.